app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".Sample2Activity">
+        <activity android:name=".SampleActivity">
```

Comment:
could you switch this back to `Sample2Activity`. That one demonstrates true NTP (vs SNTP)